src/components/misc/FeedbackCard/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { 
+  Box, 
+  Flex, 
+  Text, 
+  Badge,
+  Icon,
+} from '@chakra-ui/react'
+
+import { FaChevronUp, FaComment } from 'react-icons/fa'
+
+interface IFeedbackCardProps{
+  title: string,
+  description: string,
+  category: string,
+  votesCount: number,
+  commentsCount: number,
+}
+
+const FeedbackCard = ({title, description, category, votesCount, commentsCount}: IFeedbackCardProps) => {
+  return (
+    <Box 
+      bg='white'
+      flexDir='row'
+      display='flex'
+      flexWrap='wrap'
+      w={[327, 689, 825]} 
+      h={[200, 151, 151]} 
+      borderRadius='10px' p='24px' 
+      justifyContent='space-between'
+      sx={{
```

Comment:
Let's use here [Chakra responsive styles](https://chakra-ui.com/docs/styled-system/features/responsive-styles#the-object-syntax). I think the object syntax would be more appropriate.
That also applies to other places where you used media-queries